example/lib/shapes/shadow.dart

```diff
@@ -0,0 +1,165 @@
+import 'dart:math';
+
+import 'package:flutter/widgets.dart';
+import 'package:swift_ui/swift_ui.dart';
+
+class ShadowDemo extends StatelessWidget {
+  const ShadowDemo({
+    super.key,
+  });
+
+  static const color = Colors.grey;
+  static const radius = 10.0;
+  static const x = 10.0;
+  static const y = 10.0;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      color: Colors.white,
+      child: const VStack(
+        [
+          // Rectangle with shadow
+          Frame(
+            width: 200,
+            height: 100,
+            child: Shadow(
```

Comment:
I'm just noticing this as I read through the example code - should we get rid of the `child` named parameter in place a required positional parmeter?

I made that shift with the layout widgets, e.g., `VStack`. I did that to demonstrate to future users that you don't have to call things `children` and `child`, and that there's no magic in Swift UI to avoid those property names.

Reply:
Interesting. I'm so used to using `child` and `children` that I hadn't even thought about that before. I've removed them now. You can see what you think.